.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,39 @@
+---
+name: "\U0001F680Feature Request"
+about: How to create an issue for requesting a feature
+labels: enhancement
+title: [Feature Request] request title
+---
+
+**Important Note: We do not do technical support, nor consulting** and don't answer personal questions per email.
+Please post your question on [reddit](https://www.reddit.com/r/reinforcementlearning/) or [stack overflow](https://stackoverflow.com/) in that case.
+
+
+### 🚀 Feature
+
+A clear and concise description of the feature proposal.
+
+### Motivation
+
+Please outline the motivation for the proposal.
+Is your feature request related to a problem? e.g., I'm always frustrated when [...].
```

Comment:
Hyphens/quotation marks/etc around "I'm always..." example to clarify it is an example text for the motivation :)